torchao/quantization/qat/fake_quantize_config.py

```diff
@@ -423,6 +423,11 @@ def _infer_fake_quantize_configs(
         (act_granularity, weight_granularity) = _normalize_granularity(
             base_config.granularity
         )
+        assert act_granularity == weight_granularity and isinstance(
```

Comment:
split fake quant changes to a separate file?

Reply:
Hi @jerryzh168 This file is about fake quant. Do you mean a separate file or a separate PR?

Reply:
sorry, separate PR